src/app/modules/project-management/components/project-list/project-list.component.ts

```diff
@@ -1,23 +1,36 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { Store, select } from '@ngrx/store';
 import { Project } from '../../../shared/models';
+import { allProjects } from '../../store/project.selectors';
+import * as actions from '../../store/project.actions';
 
 @Component({
   selector: 'app-project-list',
   templateUrl: './project-list.component.html',
   styleUrls: ['./project-list.component.scss'],
 })
 export class ProjectListComponent implements OnInit {
-  @Input() projects: Project[] = [];
   @Output() editProject = new EventEmitter();
   @Output() deleteProject = new EventEmitter();
 
   projectToDelete: Project;
   openDeleteModal = false;
   filterProjects = '';
+  isLoading: boolean;
```

Comment:
I think this flag is not being used so we should not declare this field in here.

Reply:
yes, we will create an issue to include a loading in the interface, but I think this flag needs to stay here

Reply:
gotcha